pkg/reckoner/plot.go

```diff
@@ -110,16 +110,15 @@ func (c *Client) Plot() error {
 }
 
 // TemplateAll runs the same as plot but runs template instead
-func (c Client) TemplateAll() (string, error) {
-	err := c.UpdateHelmRepos()
+func (c Client) TemplateAll(templateOutputDir string) (fullOutput string, err error) {
+	err = c.UpdateHelmRepos()
 	if err != nil {
-		return "", nil
+		return fullOutput, err
 	}
 
-	var fullOutput string
 	for _, release := range c.CourseFile.Releases {
-
-		if err := c.cloneGitRepo(release); err != nil {
+		err = c.cloneGitRepo(release)
```

Comment:
Curious as to the reasoning changing this structure from the `if err := ... ;`

Reply:
just to keep the style in-line with the rest of the file (and generally speaking, the project. although there might be other instances in other files somewhere. I don't currently know)

Reply:
Gotcha. That's fine in this instance. Sometimes we switch back and forth in order to ease error handling scenarios where we may be re-using the `err` variable. Not an issue at all here, and I like being consistent. More just a heads up